arc/conformance/plugin/arc_conformance.sh

```diff
@@ -0,0 +1,236 @@
+
+  az keyvault certificate import \
+  --vault-name $keyVaultName \
+  --name ecccert1 \
+  --file testec.pfx 2> ${results_dir}/error || python3 /arc/setup_failure_handler.py
+}
+
+# validate enviorment variables
+if [ -z "${TENANT_ID}" ]; then
+  echo "ERROR: parameter TENANT_ID is required." > ${results_dir}/error
+  python3 /arc/setup_failure_handler.py
+fi
+
+if [ -z "${SUBSCRIPTION_ID}" ]; then
+  echo "ERROR: parameter SUBSCRIPTION_ID is required." > ${results_dir}/error
+  python3 /arc/setup_failure_handler.py
+fi
+
+if [ -z "${AZURE_CLIENT_ID}" ]; then
+  echo "ERROR: parameter AZURE_CLIENT_ID is required." > ${results_dir}/error
+  python3 /arc/setup_failure_handler.py
+fi
+
+if [ -z "${AZURE_CLIENT_SECRET}" ]; then
+  echo "ERROR: parameter AZURE_CLIENT_SECRET is required." > ${results_dir}/error
+  python3 /arc/setup_failure_handler.py
+fi
+
+if [ -z "${ARC_CLUSTER_NAME}" ]; then
+  echo "ERROR: parameter ARC_CLUSTER_NAME is required." > ${results_dir}/error
+  python3 /arc/setup_failure_handler.py
+fi
+
+if [ -z "${ARC_CLUSTER_RG}" ]; then
+  echo "ERROR: parameter ARC_CLUSTER_RG is required." > ${results_dir}/error
+  python3 /arc/setup_failure_handler.py
+fi
+
+# add az cli extensions 
+az extension add --name aks-preview
+az extension add --name k8s-extension
+
+# login with service principal
+az login --service-principal \
+  -u ${AZURE_CLIENT_ID} \
+  -p ${AZURE_CLIENT_SECRET} \
+  --tenant ${TENANT_ID} 2> ${results_dir}/error || python3 /arc/setup_failure_handler.py
+
+az account set --subscription ${SUBSCRIPTION_ID} 2> ${results_dir}/error || python3 /arc/setup_failure_handler.py
+
+# set environment variables
+setEnviornmentVariables
+
+# setup keyvault
+setupKeyVault
+
+# wait for resources in ARC namespace
+waitSuccessArc="$(waitForResources deployment azure-arc)"
+if [ "${waitSuccessArc}" == false ]; then
+    echo "ERROR: deployment is not avilable in namespace - azure-arc" > ${results_dir}/error
+    python3 /arc/setup_failure_handler.py
+    exit 1
+fi
+
+az k8s-extension create \
+      --name arc-akv-conformance \
+      --extension-type Microsoft.AzureKeyVaultSecretsProvider \
+      --scope cluster \
+      --cluster-name ${ARC_CLUSTER_NAME} \
+      --resource-group ${ARC_CLUSTER_RG} \
+      --cluster-type connectedClusters \
+      --release-train preview \
+      --release-namespace kube-system \
+      --configuration-settings 'secrets-store-csi-driver.enableSecretRotation=true' \
+        'secrets-store-csi-driver.rotationPollInterval=30s' \
+        'secrets-store-csi-driver.syncSecret.enabled=true'
+
+# wait for secrets store csi driver and provider pods
+kubectl wait pod -n kube-system --for=condition=Ready -l app=secrets-store-csi-driver
+kubectl wait pod -n kube-system --for=condition=Ready -l app=csi-secrets-store-provider-azure
+
+/arc/provider-e2e-test -ginkgo.v -ginkgo.dryRun=false
```

Comment:
```suggestion
/arc/provider-e2e-test -ginkgo.v
```

default is false, so we don't need to set it explicitly